src/Opserver.Web/Controllers/ExceptionsController.cs

```diff
@@ -116,6 +115,15 @@ private async Task<ExceptionStore.SearchParams> GetSearchAsync()
             return result;
         }
 
+        private DateTime? GetDate(string paramName)
+        {
+            if (GetParam(paramName) is string dateStr && DateTime.TryParse(dateStr, CultureInfo.InvariantCulture, DateTimeStyles.AssumeLocal, out var date))
```

Comment:
I've changed the CultureInfo and DateTimeStyles to match the log Creation Date format. Log creation date is not regionalized when displayed to the user so it doesn't make sense to convert the incoming dates, otherwise the datetime entered by the user and the datetime of the exception entry do not match.